<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Previous_Button">
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Next_Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Setup_Tab">
       <attribute name="title">
        <string>Setup</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>271</y>
          <width>561</width>
          <height>50</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="Project">
         <item row="1" column="1">
          <widget class="QPushButton" name="SetProjectDir_Button">
           <property name="text">
            <string>Set Directory</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="Text_ProjectDir">
           <property name="text">
            <string>Enter Path to Project Directory</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="CheckProjectDir_Button">
           <property name="text">
            <string>Check Directory</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="ProjectDir"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>561</width>
          <height>50</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="Blender">
         <item row="0" column="0">
          <widget class="QLabel" name="Text_BleDir">
           <property name="text">
            <string>Enter Path to Blender Directory</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="BleDir"/>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="SetBleDir_Button">
           <property name="text">
            <string>Set Directory</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="CheckBleDir_Button">
           <property name="text">
            <string>Check Directory</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Mesh_Tab">
       <attribute name="title">
        <string>Geometry/Mesh</string>
       </attribute>
       <widget class="QPushButton" name="Mesh_Button">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Mesh</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="Mesh_Output">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>30</y>
          <width>391</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Solver_Tab">
       <attribute name="title">
        <string>Solver</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="ARBedingungen_Tab">
       <attribute name="title">
        <string>Anfangs-/Randbedingungen</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
